Meadowlark-js/services/meadowlark-fastify/.env.example

```diff
@@ -61,12 +61,12 @@ FASTIFY_PORT=3000
 MEADOWLARK_STAGE=local
 
 #### Logging
-# Valid levels include: error, warning, info, debug
+# Valid levels include: error, warning, info, debug (higher levels provide faster logging performance)
 LOG_LEVEL=debug
-# When true, logs to plain text instead of JSON
+# When true, logs to plain text instead of JSON (slower logging performance)
 IS_LOCAL=true
-# By default, it will not save to a log file, set to true if desired
-# SAVE_LOG_TO_FILE=false
+# By default, it writes logs to STDOUT. Set to true to log to a file instead (faster logging performance)
+# LOG_TO_FILE=false
```

Comment:
Changed `SAVE_LOG_TO_FILE` to `LOG_TO_FILE` to be more in line with `LOG_LEVEL` and `LOG_FILE_LOCATION`